core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.ts

```diff
@@ -16,20 +16,17 @@
  * @fileoverview Directive for the activities tab in the admin panel when Oppia
  * is in production mode.
  */
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-require('domain/utilities/url-interpolation.service.ts');
-
-angular.module('oppia').directive('adminProdModeActivitiesTab', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      bindToController: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/admin-page/activities-tab/' +
-        'admin-prod-mode-activities-tab.directive.html'),
-      controllerAs: '$ctrl',
-      controller: [function() {}]
-    };
-  }
-]);
+@Component({
+  selector: 'admin-prod-mode-activities-tab',
+  templateUrl: './admin-prod-mode-activities-tab.directive.html',
+  styleUrls: []
+})
+export class AdminProdModeActivitiesTabComponent implements OnInit {
+  ngOnInit(): void {}
```

Comment:
Do we need to define this empty function?

Reply:
@Nik-09, is this comment resolved? [Make sure to reply to all the comments as asked on [the wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me).]

Reply:
Sorry by accident, I marked resolved instead of comment..
btw now I have removed that empty function. PTAL